### PR TITLE
Flow typings for pirates

### PR DESCRIPTION
**Pull request by AGawrys**
# ↪️ Pull Request

This PR is for flow-typing for `pirates` 

Source(s): https://github.com/ariporad/pirates
## 🚨 Test instructions

Removed `// flowlint-next-line untyped-import:off` from `register.js`

### CODE REVIEW

File: flow-libs/pirates.js.flow [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// @flow
+
+// Derived from : https://github.com/ariporad/pirates/blob/master/index.d.ts
+// Which is licensed MIT.
+
+interface Options {
```
Comment (mischnic) [inline flow-libs/pirates.js.flow:6]: Flow interfaces allow passing other (unlisted) properties. So this should probably be `type Options = {| `

https://flow.org/try/#0JYOwLgpgTgZghgYwgAgPIAczAPYgM7IDeAUMshAB5h4D8AXMgIJRRwCeAPHmFKAOYA+ANylkwPiGxQIAOWwATCAFkFAVwA2EWgwBG2bJrggRAX2LEYqkAiy5kMABTZMeBhlv4AlETMWHhPGwAWwgwAAt+AFF1PAgGAG0AXRNPISA

**Comment by AGawrys** [comment]
@wbinnssmith updated this PR by resolving the comment

File: flow-libs/pirates.js.flow [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// @flow
+
+// Derived from : https://github.com/ariporad/pirates/blob/master/index.d.ts
+// Which is licensed MIT.
+type Options = {|
+  matcher?: Matcher,
+  exts?: Array<string>,
+  ignoreNodeModules?: boolean,
+|};
```
Comment (mischnic) [inline flow-libs/pirates.js.flow:9]: Move this into the `declare module` block. Otherwise it's a global type

Comment (AGawrys) [reply flow-libs/pirates.js.flow]: ok updated this 🙇  @mischnic

**Review by mischnic — APPROVED** [review]

**Pull request merged**